### DISCUSSION TITLE
LilyGO T-TWR Plus V2.1

### DISCUSSION THREAD

**Discussion by lyusupov** [nakhonthai/ESP32APRS_T-TWR#26] [General]
https://twitter.com/lilygo9/status/1752865846727446786

**Comment by lyusupov** [comment]
![](https://raw.githubusercontent.com/lyusupov/SoftRF/master/documents/images/ham-29.jpg)

<br>
<br>

![](https://raw.githubusercontent.com/lyusupov/SoftRF/master/documents/images/ham-28.jpg)

**Comment by lyusupov** [comment]
[**These are the differences between Rev2.1 and Rev2.0**](https://github.com/Xinyuan-LilyGO/T-TWR#there-are-the-following-differences-between-rev21-and-rev20)

**Comment by lyusupov** [comment]
[**T-TWR V2.1 is in the Store !**](https://www.lilygo.cc/products/t-twr-rev2-1)

![](https://www.lilygo.cc/cdn/shop/files/H709-T-TWRREV2_5.jpg)

**Comment by nakhonthai** [comment]
Perfectly, I'm waiting for a donation to be used to develop the firmware.

**Reply by TeddJohnson** [reply]
Do you have venmo/paypal/some way that we could send money to help facilitate this?

while I don't really wanna drop $60 to donate the whole board (especially since I still have to buy mine), I would be willing to donate $10. And I bet there's at least 6 of us who would do that

**Reply by derekmurawsky** [reply]
I'm in for $30 if you're willing to support the platform.

**Comment by sethpark777** [comment]
Is there anything that you think would prevent the current firmware from working with the V2.1? I just got one and would be willing to do whatever testing or help however I can.

**Comment by sethpark777** [comment]
I made these measurements so figured I would post them here in case they were useful in any way. Measurements were taken with a high quality 21700 at 3.9v. 

![PXL_20240822_014845174~2](https://github.com/user-attachments/assets/bd188f08-e66f-4560-ae11-470cf69447f2)
![PXL_20240822_014053313~2](https://github.com/user-attachments/assets/8e2232f9-09b0-4fc7-938a-779c040fe586)

**Comment by Symo** [comment]
I now have a V2.0 and V2.1 T-TWR.  The V2.0 is already esp32aprs!!  There has to be more interest than the 4 of us in this thread for a version on the V2.1 hardware! ;-)

**Comment by pagex** [comment]
2.1 seems not called Plus and a bit cheaper

**Comment by soulthatcreates** [comment]
Hello there,
Is there any update on this? Does it work with the 2.1 at all?

**Comment by guido57** [comment]
I did a few modifications to get it working on T-TWR Version 2.1
See https://github.com/guido57/APRS_T-TWR2_1
Guido IW5ALZ
